modules/mesh-task/templates/consul_client_command.tpl

```diff
@@ -11,6 +11,7 @@ exec consul agent \
   -advertise "$ECS_IPV4" \
   -data-dir /consul/data \
   -client 0.0.0.0 \
+  -encrypt "$CONSUL_GOSSIP_ENCRYPTION_KEY" \
```

Comment:
If `CONSUL_GOSSIP_ENCRYPTION_KEY` is empty, it passes an empty key (`-encrypt ""`) and Consul starts up with gossip encryption disabled. So no template conditional needed.

Reply:
Sure! I added a conditional for the `-encrypt` option.